### PR TITLE
Migrate dasboards to grafana 10 2

### PR DESCRIPTION
**Pull request by SlimaneAmar**

### CODE REVIEW

File: docker-compose/technical/docker-compose.technical.yml [hunk 1]
```diff
@@ -127,7 +127,7 @@ services:
     profiles:
       - all
       - metrics
-    image: grafana/grafana:8.2.3
+    image: grafana/grafana:10.2.2
```
Comment (antoinebhs) [inline docker-compose/technical/docker-compose.technical.yml:130]: No update of prometheus?

Comment (SlimaneAmar) [reply docker-compose/technical/docker-compose.technical.yml:130]: Jimmy said it was the same version

**Review by antoinebhs — CHANGES_REQUESTED** [review]
* On devopf, there is no data for security analysis in the **Computations** dashboard
* Should remove the "Network load time" as it's not meaningful in the **Computations** dashboard
* Need to handle multiple pods in the **Computations** dashboard (and maybe network-conversion?), it gets messy otherwise: 
image

**Review by antoinebhs — APPROVED** [review]
Comments above to fix in separate PR

**Pull request merged**